circus-train-api/src/main/java/com/hotels/bdp/circustrain/api/data/DataManipulator.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2016-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.api.data;
+
+import java.io.IOException;
+
+public interface DataManipulator {
+
+  /**
+   * Returns boolean stating whether the deletion was successful or not. May through an error while attempting to delete
```

Comment:
Please document the "may throw an error" part using the standard Javadoc for exceptions thrown.